### PR TITLE
GPU-994 - Java 17 Update

### PR DESCRIPTION
**Pull request by guggi**
### General

* [ ] Added abstract that describes the change
* [ ] Added changelog entry to `/CHANGELOG.adoc`
* [ ] Ensured that the change is covered by tests
* [ ] Ensured that the change is documented in the docs

### On API Changes

* [ ] Checked if the changes are breaking or not
* [ ] Added GraphQL API if applicable
* [ ] Added Elasticsearch mapping if applicable

### CODE REVIEW

File: .jenkins/Dockerfile [hunk 1]
```diff
@@ -33,14 +33,14 @@ RUN curl "https://get.docker.com/builds/`uname -s`/`uname -m`/docker-latest.tgz"
     mv docker/* /usr/bin &&\
     rm -rf docke*
 
-# Install Java 11 (default)
+# Install Java 11
```
Comment (guggi) [inline .jenkins/Dockerfile:36]: are old versions still needed or are we safe to remove alternative java version?

File: tests/pom.xml [hunk 2]
```diff
@@ -126,9 +126,23 @@
...
+					<compilerVersion>1.8</compilerVersion>
+					<source>17</source>
+					<target>17</target>
+				</configuration>
+			</plugin>
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-surefire-plugin</artifactId>
+				<configuration>
+					<argLine>--add-opens=java.base/java.lang=ALL-UNNAMED ${jacoco.env}</argLine>
```
Comment (guggi) [inline tests/pom.xml:144]: todo: check if needed

**Review by npomaroli — CHANGES_REQUESTED** [review]

File: tests/pom.xml [hunk 3]
```diff
@@ -126,9 +126,21 @@
 
 	<build>
 		<plugins>
+			<plugin>
```
[the remark this answers is not in the document]
Comment (guggi) [reply tests/pom.xml:128]: [c98f0a7](https://github.com/gentics/mesh/pull/1558/commits/c98f0a7a9c92ab494da9285cb9c540646e3fe936)

File: pom.xml [hunk 4]
```diff
@@ -217,11 +217,10 @@
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-compiler-plugin</artifactId>
+				<version>3.11.0</version>
```
[the remark this answers is not in the document]
Comment (guggi) [reply pom.xml]: [c98f0a7](https://github.com/gentics/mesh/pull/1558/commits/c98f0a7a9c92ab494da9285cb9c540646e3fe936)

**Pull request merged**